### PR TITLE
Add Dutch localisation

### PR DESCRIPTION
**Pull request by Bartwillemsen**
Translated template file to Dutch.

I hope I did this right :stuck_out_tongue: If there are any problems, please let me know and I will fix it.

### CODE REVIEW

File: locale/nl/BingWallpaper.po [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+msgid "Bing Wallpaper of the Day for"
+msgstr "Bing achtergrond van de dag voor"
+
+#: extension.js:295
+msgid "No wallpaper available"
+msgstr "Geen achtergrond beschikbaar"
+
+#: extension.js:296
+msgid "No picture for today 😞."
+msgstr "No afbeelding voor vandaag 😞."
```
Comment (lasermarty) [inline locale/nl/BingWallpaper.po]: You forgot to translate 'No'. "Geen afbeelding voor vandaag 😞." would be correct here.

File: locale/nl/BingWallpaper.po [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+#: Settings.ui.h:21
+msgid ""
+"<span size=\"small\">This program comes with ABSOLUTELY NO WARRANTY.\n"
+"See the <a href=\"https://www.gnu.org/licenses/old-licenses/gpl-2.0.html"
+"\">GNU General Public License, version 3 or later</a> for details.</span>"
+msgstr ""
+"<span size=\"small\">Dit programma heeft ABSOLUUT GEEN GARANTIE.\n"
+"Zie de <a href=\"https://www.gnu.org/licenses/old-licenses/gpl-2.0.html"
+"\">GNU General Public License, versie 3 or nieuwer</a> voor details.</span>"
```
Comment (lasermarty) [inline locale/nl/BingWallpaper.po]: 'versie 3 **or** nieuwer' should be 'versie 3 **of** nieuwer'.

File: locale/nl/BingWallpaper.po [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+"jouw regio. Toont een notificatie voor de afbeeldingsbeschrijving en "
+"copyright informatie."
+
+#: Settings.ui.h:18
+msgid "Maintained by Michael Carroll"
+msgstr "Onderhouden door Michael Carroll"
+
+#: Settings.ui.h:19
+msgid "Based on NASA APOD Gnome shell extension by Elia Argentieri"
+msgstr "Gebasseerd op de NASA APOD Gnome Shell extentie door Elia Argentieri"
```
Comment (lasermarty) [inline locale/nl/BingWallpaper.po]: Gebaseerd*

File: locale/nl/BingWallpaper.po [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+msgid "Default is \"auto\""
+msgstr "Standaard is \"auto\""
+
+#: Settings.ui.h:15 extension.js:131
+msgid "Settings"
+msgstr "Instellingen"
+
+#: Settings.ui.h:16
+msgid "Gnome shell extension version: "
+msgstr "Gnome shell extentie versie:"
```
Comment (lasermarty) [inline locale/nl/BingWallpaper.po]: extensie*

**Comment by Bartwillemsen** [comment]
Thanks @lasermarty I fixed your comments :)

**Comment by neffo** [comment]
Ahh, thanks!

_dank je wel_

**Pull request merged**